### PR TITLE
Update linters @v3

### LINKED ISSUE #1043: Run linters for PR

### PR DESCRIPTION
**Pull request by qfox**
Ref #1235 
Closes #1043

### CODE REVIEW

**Comment by narqo** [comment]
Any ideas what to do with peerDependencies? Maybe we should use npm@2x?

**Comment by qfox** [comment]
@narqo Don't get why we should. If we have some magic with peerDeps we should just add that deps explicitly.

**Comment by narqo** [comment]
Sounds reasonable, but unfortunately I'm not the person who can answer what is going on with `enb-*` now. So help wanted.

**Comment by dfilatov** [comment]
@blond ping

**Comment by dfilatov** [comment]
As @blond is busy, @zxqfox could you do it by yourself?

**Comment by qfox** [comment]
@blond Not sure I can do this correctly. I'll give it a try tomorrow.

File: package.json [hunk 1]
```diff
@@ -14,17 +14,18 @@
   "author": "Sergey Belov <peimei@ya.ru>",
   "license": "MPL-2.0",
   "dependencies": {
-    "ym": "0.1.0",
+    "bem-environ": "~1.4.0",
     "bem-xjst": "0.9.0",
     "bemhtml-compat": "~0.1.2",
-    "bem-environ": "~1.4.0"
+    "jshint-groups": "^0.8.0",
```
Comment (narqo) [inline package.json:20]: Looks like it must be in `devDependencies`, isn't it?

**Comment by dfilatov** [comment]
@zxqfox After #1276 everything should be fine with npm. Could you check?

**Comment by veged** [comment]
rebase on top of current `v4` please (see #1239)

**Pull request closed without merging**